crates/primitives/src/identity.rs

```diff
@@ -90,6 +90,10 @@ pub enum WalletType {
         #[serde(rename = "chainId")]
         chain_id: u64,
     },
+    STARKNET {
+        #[serde(rename = "walletName")]
```

Comment:
can we make this

```suggestion
    #[serde(rename_all = "camelCase")]
    STARKNET {
```

we can apply the same to `ETH`

Reply:
here we cannot use rename_all since it is designed to be used on containers, not fields. Since we already have rename_all = "UPPERCASE" on WalletType I would leave it like it is now.